### ISSUE TITLE
[Bug]: Streepass relay causes issues with Tomodachi Life

### ISSUE THREAD

**Issue by mrjvs** [PretendoNetwork/BOSS#14]
### Checked Existing

- [X] I have checked the repository for duplicate issues.

### What happened?

There's a streetpass mechanic in Tomodachi Life where you can share travelers, which are the sons and daughters of the married couples of your island. 
When you streetpass with someone else and you both have this game, the first traveler in your queue will be sent to their island, and the first traveler in their queue will travel to your island.

However, when using the streetpass relay function of pretendo, many times the traveler will not appear in the other person's island, making it disappear forever.

This is a serious issue for the streetpass functionality of this game, because:
1. You cannot recover those travelers in any way, once they disappear due to this bug they will remain lost forever.
2. You cannot turn off the streetpass relay function on pretendo, so there's no way to evade this bug while using the pretendo service.
3. This bug also affects people who don't have pretendo installed, because if you have recieved a traveler born in the island of another person which doesn't have pretendo and you send that traveler through streetpass relay, it's possible this bug may happen, thus making disappear a traveler from someone who doesn't use this service.

### What did you expect to happen?

This doesn't happen in when doing a regular streetpass between consoles.

### Other relevant information. (OPTIONAL)

Basically, this bug fundamentally breaks the whole travelers mechanic of this game, and it seriously affects every person who streetpass using this game, whether they use pretendo or not.

---

Patched together from [a bug report ](https://discord.com/channels/408718485913468928/1320436576169230471/1320436576169230471) on support channel on the discord.

**Comment by jonbarrow** [comment]
Moved to the correct repo

**Comment by mrjvs** [comment]
Thanks, me and the mod team were unsure where streetpass resided.

**Comment by jonbarrow** [comment]
I'm not sure there's anything we can do about this? Outside of just blocking SPR for this title? But that feels a bit hacky. Does this also happen on NN?

We don't really do much in terms of SPR, it's mostly an on-console thing. All we do is store the data the console tells us to store and then distribute it like normal. So if the game can't cope with SPR I'm not sure how in-scope this is for us?

Will tag @DaniElectra and @shutterbug2000 though since Dani knows more about SPR than I do and Shutter (iirc) knows about Tomodachi Life specifically

**Comment by tofu211** [comment]
Hello! I'm the original poster of this issue in pretendo's discord server. I'd like to thank @mrjvs again for opening this thread for me.
I'd like to share some more thoughts on this problem.

I've searched around old forums and i've found that people were complaining about issues with streetpass relay back in the day, and the apparent solution to that was to ensure tomodachi life isn't at the very first place of the streetpass data menu for some reason...?

Also, one thing i've noticed is that this game's streetpass mechanics are tied with spotpass, for example, sometimes your traveller would come back after travelling for a while through spotpass to your house, or they would ocasionally send you letters that way. However, the abscense of spotpass in pretendo's services shouldn't cause this bug at all, unless there's some kind of check whenever a streetpass tag happens to ensure your traveller doesn't get lost due to possible errors within streetpass relay's functioning. Basically, nintendo may have been aware of these kinds of errors with streetpass relay, so they could have used spotpass as a way to ensure nothing weird happens, but idk.

This is all the information I managed to find... I hope it's somewhat helpful to you :)

**Comment by tofu211** [comment]
https://gamefaqs.gamespot.com/boards/643013-tomodachi-life/69625836?page=1
https://youtu.be/FY2AThaOz8c

This is some of the kind of stuff i've found when searching for "traveler bug". The quick fix of this is possibly not accurate but it's at least proof people had problems with streetpass relay back in the day.

**Comment by tofu211** [comment]
For now though, I think the best solution would be what @jonbarrow proposed. Blocking SPR for this title while this issue is being adressed to ensure no more travellers disappear.
